.github/workflows/hello.yml

```
@@ -0,0 +1,20 @@
+name: hello
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
```

What happens if you change this:
```suggestion
on: [push]
```